src/openforms/templates/cookie_consent/cookiegroup_list.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'ui/views/abstract/list.html' %}
+{% load i18n %}
+
+
+{% block card %}
+    <header class="card__header">
+        <h1 class="title">{% trans "Cookies" %}</h1>
+    </header>
+
+    <div class="card__body">
+        {% block card_body %}
+            <p>
+              {% trans "This is a list of the categories of cookies used in our website and why we use them." %}
+            </p>
+            <ul class="list">
+                {% for cookie_group in object_list  %}
+                    <li class="list__item">
+                        {% include "cookie_consent/_cookie_group.html" %}
+                    </li>
+                {% endfor %}
+            </ul>
+        {% endblock %}
+    </div>
+{% endblock %}
```

Comment:
I think this page needs a "Close"-button that will take the user back to the previous page. After selecting Accept/Decline, there's not web-way to return to where I was when I clicked the cookiebanner message.